### DISCUSSION TITLE
Is it possible to display the folder size in status bar?

### DISCUSSION THREAD

**Discussion by UtkarshVerma** [sxyazi/yazi#661] [1. Q&A] [answered]
The status bar has a section where the size of file under cursor is displayed. Is it possible for it to display the folder size?

![image](https://github.com/sxyazi/yazi/assets/31820255/ad517a85-453a-46b5-aa20-b328e6741153)

**Comment by sxyazi** [comment]
see https://github.com/sxyazi/yazi/discussions/492#discussioncomment-8061404

**Reply by UtkarshVerma** [reply, marked as answer]
Thank you. That works as expected.
